components/user_settings/security/mfa_section/mfa_section.test.tsx

```diff
@@ -101,21 +101,33 @@ describe('MfaSection', () => {
 
     describe('setupMfa', () => {
         it('should send to setup page', () => {
-            const wrapper = mountWithIntl(<MfaSection {...baseProps}/>);
```

Comment:
Why did you remove `mountWithIntl`?

Reply:
In or der to fix the error `Property 'setupMfa' does not exist on type 'Component<{}, {}, any>'.` then I switch to `ShallowWrapper`

Reply:
Could you try if making explicit use of the generic solves that problem? What I mean is something like:
`mountWithIntl<MfaSection>(<MfaSection {...baseProps}/>);`

Reply:
already tried but still got an error:
```
Type 'MfaSection' does not satisfy the constraint 'ReactElement<any, string | ((props: any) => ReactElement<any, string | ... | (new (props: any) => Component<any, any, any>)> | null) | (new (props: any) => Component<any, any, any>)> | IntlInjectedElement'.
  Type 'MfaSection' is missing the following properties from type 'ReactElement<any, ForwardRefExoticComponent<Pick<WithIntlProps<WrappedComponentProps<any>>, any> & RefAttributes<ComponentType<any>>> & { ...; }>': type, key

(alias) class MfaSection
import MfaSection
```

Reply:
You are right. The right way to do it would be using this line instead:
`(wrapper.instance() as MfaSection).setupMfa(mockEvent);`

Sorry for the wrong lead before.